.rubocop.yml

```diff
@@ -12,4 +12,6 @@ Metrics/ClassLength:
 
 Lint/AssignmentInCondition:
     Exclude: 
-      - 'bin/spring'
\ No newline at end of file
+      - 'bin/spring'
+      - 'db/migrate'
+      - 'db/schema.rb'
```

Comment:
@abapolov this is a wrong place for this file. There are still `60 offenses`.
Please run `rubocop` before next commit :wink: